src/GameState.py

```diff
@@ -15,4 +15,7 @@ def ApplyAction(self, action):
         raise NotImplementedError
     
     def Winner(self, prevAction = None):
+        raise NotImplementedError
+
+    def NumericRepresentation(self):
```

Comment:
MCTS doesn't care that there is a NumericRepresentation. I don't think this needs to be in here.

Reply:
<a href="https://github.com/jordan-singer/BlackBird/commit/7bb29fd52defe03073b6d7610022596d158f61c9">Resolved</a>.